src/gt4py/backend/pyext_builder.py

```diff
@@ -55,7 +55,9 @@ def get_gt_pyext_build_opts(
         cuda_arch = gt_config.build_settings["cuda_arch"] or compute_capability
         if not cuda_arch:
             raise RuntimeError("CUDA architecture could not be determined")
-        elif compute_capability and int(compute_capability) < int(cuda_arch):
+        elif cuda_arch.startswith("sm_"):
```

Comment:
```suggestion
        if cuda_arch.startswith("sm_"):
```
Just nitpicking, but the two `if`s are testing for different concepts (validity <-> format).

Reply:
Change made, thank you.